interface/billing/billing_process.php

```diff
@@ -20,10 +20,10 @@
 require_once("../globals.php");
 require_once("$srcdir/patient.inc");
 require_once("$srcdir/billrep.inc");
-require_once("$srcdir/billing.inc");
 
 use OpenEMR\Billing\X12_5010_837P;
 use OpenEMR\Billing\HCFA_1500;
+use OpenEMR\Billing\Billing;
```

Comment:
probably would make the class name more specific (to avoid clashes in future when have more namespacing being used. Maybe something like `BillingUtilities` . Also should keep the `use` ordered alphabetically so easier in future when we have more of them.

Reply:
ok thanks